lib/simulator-xcode-9.js

```diff
@@ -212,12 +212,19 @@ class SimulatorXcode9 extends SimulatorXcode8 {
       newPrefs.DevicePreferences = {[this.udid.toUpperCase()]: devicePrefs};
     }
     newPrefs = _.merge(newPrefs, commonPrefs);
+    let isUpdateSuccessful = true;
     await preferencesPlistGuard.acquire(SimulatorXcode9.name, async () => {
```

Comment:
The return value of the passed-in function is returned by `preferencesPlistGuard.acquire` so this can be simplified to 
```js
return await preferencesPlistGuard.acquire(SimulatorXcode9.name, async () => {
  try {
    const currentPlistContent = await plist.parsePlistFile(plistPath);
    await plist.updatePlistFile(plistPath, _.merge(currentPlistContent, newPrefs), true);
    log.debug(`Updated ${this.udid} Simulator preferences at '${plistPath}' with ${JSON.stringify(newPrefs)}`);
    return true;
  } catch (e) {
    log.warn(`Cannot update ${this.udid} Simulator preferences at '${plistPath}'. ` +
             `Try to delete the file manually in order to reset it. Original error: ${e.message}`);
    return false;
  }
});
```

Reply:
thanks, fixed